Makefile.kubebuilder

```diff
@@ -0,0 +1,129 @@
+# Copyright 2019 The Kanister Authors.
+#
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Image URL to use all building/pushing image targets
+
+REPOSITORY_SERVER_IMAGE_NAME := repo-server-controller
+IMG ?= $(REGISTRY)/$(REPOSITORY_SERVER_IMAGE_NAME)
+
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.25.0
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
```

Comment:
We should use the same directory path as the Kanister controller binary which is defined here - https://github.com/kanisterio/kanister/blob/master/Makefile#L97. Instead of GOBIN, set a custom path to put built binary.